scripts/qa_crowbarsetup.sh

```diff
@@ -2059,22 +2058,26 @@ function reboot_nodes_via_ipmi()
             local ip=$bmc_net.$(($ip4 + $i))
             if [ $i -gt $nodenumber ]; then
                 # power off extra nodes
-                ipmitool -H $ip -U root -P $pw power off &
+                ipmitool -H $ip -U root -P $pw power off
+                wait_for 2 60 "ipmitool -H $ip -U root -P $pw power status | grep -q 'is off'" "node to power off"
             else
                 ping -c 3 $ip > /dev/null || {
                     echo "error: BMC $ip is not reachable!"
                 }
-                (
+
                 ipmitool -H $ip -U root -P $pw lan set 1 defgw ipaddr "${bmc_values[1]}"
-                sleep $((5 + RANDOM % 5))
+                sleep $((50 + RANDOM % 20))
```

Comment:
This is inexpiable. It was found in an empirical way. Additional explanation could be Bernards comment a bit higher up the list: "in the IPMI tests I did, I saw the magic time of 35 seconds during which the IPMI interface is away, so the old one was too short most of the time while the new one is always long enough."